tools/clang/unittests/SPIRV/CodeGenSPIRVTest.cpp

```diff
@@ -420,4 +420,12 @@ TEST_F(FileTest, VulkanExplicitBindingReassigned) {
   runFileTest("vk.binding.explicit.error.hlsl", /*expectSuccess*/ false);
 }
 
+// For compute shaders
+TEST_F(FileTest, ComputeNumThreadsAttr) {
+  runFileTest("attribute.numthreads.hlsl");
+}
+TEST_F(FileTest, ComputeMissingNumThreadsAttr) {
+  runFileTest("attribute.missing.numthreads.hlsl");
```

Comment:
Sorry. Please bear with me for the last nit. We are naming test cases with "bigger-scope.smaller-scope" scheme right now. So IMHO this should be named as `attribute.numthreads.missing.hlsl`. :)